src/test/java/org/reaktivity/nukleus/http_cache/internal/streams/proxy/EdgeArchProxyIT.java

```diff
@@ -239,4 +248,44 @@ public void shouldCancelPushPromisesOn403() throws Exception
         k3po.finish();
         counters.assertExpectedCacheEntries(1);
     }
+
+    @Test
+    @Specification({
+        "${route}/proxy/controller",
+        "${streams}/not.use.freshness.ext.in.validation.if.not.polling/accept/client",
+        "${streams}/not.use.freshness.ext.in.validation.if.not.polling/connect/server",
+    })
+    public void shouldNotUseFreshnessExtInValidationIfNotPolling() throws Exception
```

Comment:
This test failed for me when I ran just now:
```
  Run 1: EdgeArchProxyIT.shouldNotUseFreshnessExtInValidationIfNotPolling Specified behavior did not match expected:<...
[${http:header("content-length", "19")}
                       ${http:header("date", response3Date)}

but was

[[0x00 0x0e 0x63 0x6f 0x6e 0x74 0x65 0x6e 0x74 0x2d 0x6c 0x65 0x6e 0x67 0x74 0x68 0x02 0x00 0x31 0x31

```
That hex is content-length: 11, so issue was reading that instead of content-length: 19